pkg/apis/openebs.io/scp/v1alpha1/capabilties.go

```diff
@@ -45,19 +44,19 @@ type DataStorageCapabilities struct {
 	AccessModes []AccessMode `json:"accessModes"`
 
 	// ProvisioningPolicy defines provisioning policy type, viz thick, thin
+	// +optional
 	ProvisioningPolicy []ProvisioningPolicy `json:"provisioningPolicy"`
 
 	// MultiPathing to be supported or not.
-	// +optional
-	MultiPathing []string `json:"multiPathing,omitempty"`
+	MultiPathing []MultiPathing `json:"multiPathing,omitempty"`
```

Comment:
Lets retain the optional flag.
multipathing = none should be treated the same as no multipath